frontend/compositions/officer-view/officer-work-history/index.tsx

```diff
@@ -0,0 +1,22 @@
+import { OfficerRecordType } from "../../../models/officer"
+import WorkHistoryInstance from "./work-history-instance"
+import styles from "./officer-work-history.module.css"
+
+export default function OfficerWorkHistory(officer: OfficerRecordType) {
+  const { workHistory } = officer
+  const { category, workInstanceBlock } = styles
+
+  let result: JSX.Element[] = []
```

Comment:
you can make this into a const still since not changing the object but pushing into it. for a more declarative way:
`const result = workHistory.map( (item, index) => <WorkHistoryInstance key={index + 'workHistoryItem'} {...item} />)`